### ISSUE TITLE
Latest iGPU drvs add IntelGPUCompiler OCL SPIR-V support.. Updated IntelGPUCompiler headers &ShaderAnalyzer OCL support?? 

### ISSUE THREAD

**Issue by oscarbg** [GameTechDev/IntelShaderAnalyzer#2]
Hi,
just seen latest public graphics driver  26.20.100.7000:
https://downloadcenter.intel.com/download/28991/Intel-Graphics-Windows-10-DCH-Drivers?product=88345
mentions:
"Added OpenCL SPIR-V support to the public IntelGPUCompiler API on top of existing DX11 and DX12 support. IntelGPUCompiler API allows developers to compile their shaders into Intel GPU assembly instructions."
any way to obtain updated IntelGPUCompiler headers for OpenCL support?
better yet IntelShaderAnalyzer OpenCL support?
thanks..

**Comment by JeffersonMontgomery-Intel** [comment]
Yes, good catch :)  I have an update for this project but unfortunately I'm travelling and can't push it until early next week.

**Comment by oscarbg** [comment]
nice to know.. can wait.. not urgent..
thanks

**Comment by oscarbg** [comment]
any update?

**Comment by DaveBookout-Intel** [comment]
Sorry, the original plan was to release the IntelGPUCompiler with the driver, but this has not happened. Contrary to the release notes you quote, it wasn't actually made public. I should have an update next week.

**Comment by oscarbg** [comment]
thanks for the update Dave..

**Comment by DaveBookout-Intel** [comment]
We will not be supporting OpenCL at this time. I hope to post an updated binary for DirectX/HLSL support, but I do not have an estimated delivery date for that at this time. Current focus is on the DPC++ and oneAPI-based tools. https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-programming-model/data-parallel-c-dpc.html

**Comment by oscarbg** [comment]
@DaveBookout-Intel OK, thanks for the updated info.. not familiar with DPC++ & OneAPI stack, but curious, it has tools to inspect generated Intel GPU ISA from the compute kernels?
